processing/data_collection/gazette/settings.py

```diff
@@ -9,4 +9,4 @@
     'gazette.pipelines.PdfParsingPipeline': 200,
     'gazette.pipelines.PostgreSQLPipeline': 300,
 }
-FILES_STORE = '/mnt/data/'
+FILES_STORE = '/mnt/files/'
```

Comment:
Why move it to a different folder?

Reply:
Because having only /mnt/data storing territories.csv and full directory (that contains PDFs and text files) is not possible, due to permission errors.
If we create the volume to /mnt/data, there's no territories.csv available to the stage 'make setup'.
Creating a separate volume to store the PDFs and text files, we still have territories.csv available to populating the database e files scraped without permission errors.

Reply:
> If we create the volume to /mnt/data, there's no territories.csv available to the stage 'make setup'.
You can you the [COPY instruction](https://docs.docker.com/engine/reference/builder/#copy) to make it available.

Reply:
Good point! I'm refactoring...